rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -537,6 +606,15 @@ void log_frame::UpdateUI()
 		if (steady_clock::now() >= start + 4ms || buf.empty()) break;
 	}
 
+	if (!m_log && !m_plain_log)
+	{
+		return;
+	}
+
+	const auto font_start_tag = [](const QColor& color) -> const QString { return "<font color = \"" + color.name() + "\">"; };
```

Comment:
```suggestion
	const auto font_start_tag = [](const QColor& color) -> const QString { return "<font color = \"" % color.name() % "\">"; };
```

This is a really good use case for `QStringBuilder`. Its `operator %` has overloads for `const char[N]` so I don't think further annotation is needed for those here.